andela/ah-legion-frontend
# #164046205 Setup React Router Version 4

### What does this PR do?
It setups react router version 4.

### Description
Installs and configures the router by creating these folders: components, routes and css and these components: Header, Footer, Login and Home  in order to create a dummy route for login and the "/" page to test the configuration.

**Type of change**
- [x] chore(non-breaking change that sets up react router).

### Change log
- App.scss and index.scss files have been moved to the `css` folder
- App.js has been removed and replaced with `routes/index.js``

### How to manually test this
Clone the repo and checkout to this branch `ch-react-router-164046205`. Install the extra dependencies: `react-router-dom` and `react-bootstrap`  then start the app using `npm start`. On the browser you should see the navbar, a `Home` text and the footer. On the navbar, click `Login` to go to the Login component.

### PT
[#164046205](https://www.pivotaltracker.com/story/show/164046201)
